runtime/doc/options.txt

```diff
@@ -3267,7 +3267,7 @@ A jump table for the options with a short description can be found at |Q_op|.
 				     >:SignColumn,B:SpellBad,P:SpellCap,
 				     R:SpellRare,L:SpellLocal,-:Conceal,
 				     +:Pmenu,=:PmenuSel,x:PmenuSbar,
-				     X:PmenuThumb")
+				     X:PmenuThumb,q:QuickFixLine")
```

Comment:
and I agree `snapshot_util()` is poorly documented. Unlike most other helper functions it has the unfortunate property of being completely non-self-documenting as it never appears in actual test code. It could be added to whatever dev doc or wiki page people might or might not read, but maybe the `expect()` error output should remain a little bit cumbersome just to have an excuse for mentioning `snapshot_util()` in the error message..